### DISCUSSION TITLE
We are excited to share the hackathon winners! 🎉

### DISCUSSION THREAD

**Discussion by aycabas** [microsoft/hack-together-teams#147] [Announcements]
* 🏆 Grand prize winner: [Live patient review](https://github.com/microsoft/hack-together-teams/issues/105)
* 🏅 Best AI-powered solution: [Knowledge Quest Teams Bot](https://github.com/microsoft/hack-together-teams/issues/78)
* 🏅 Best productivity-focused solution: [Guest user overview in teams](https://github.com/microsoft/hack-together-teams/issues/69)
* 🏅 Community hack winner: [DocAssistant](https://github.com/microsoft/hack-together-teams/issues/112)

>⭐ Special jury picks: Since we’ve got so many cool submissions, we decided to recognize three more projects with a special jury prize!
>  * [GeoQuest](https://github.com/microsoft/hack-together-teams/issues/104)
>  * [MsTeams Quiz Bot](https://github.com/microsoft/hack-together-teams/issues/89)
>  * [NexusAI](https://github.com/microsoft/hack-together-teams/issues/110)

**Congratulations to the winners and a huge thank you to all participants!**

For more details, check out the [announcement article](https://devblogs.microsoft.com/microsoft365dev/announcing-the-hacktogether-microsoft-teams-global-hack-winners/#%f0%9f%8f%86-grand-prize-winner-live-patient-review).

**Comment by reshmee011** [comment]
Congrats to all the winners, amazing solutions.

**Comment by garrytrinder** [comment]
Thank you to everyone who took part and submitted projects.

Congratulations to our winners!

**Comment by rabwill** [comment]
Congratulations everyone 👏🏾👏🏾👏🏾

**Comment by VickyTaar** [comment]
Congratulations to all the winners
